src/host/polyfills/thirdPartyI18n.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export default function applyThirdPartyI18nLocalesPatch(content: string) {
+    const pattern = /const\s*locales\s*=\s*\{[^]+?};/g;

```

Comment:
Let´s leave a comment here so we remember to remove this patch when we implement localization.

Reply:
done.